### PR TITLE
[webdriver] add test for cookie same site property

### PR DESCRIPTION
**Pull request by christian-bromann**
As per spec change in: https://github.com/w3c/webdriver/pull/1473

### CODE REVIEW

**Review by andreastt — CHANGES_REQUESTED** [review]
We should ideally test that lax- and strict cookies behave as expected when set.

I think we can test same-site cookies using the [predefined domains available from wptserve](https://web-platform-tests.org/writing-tests/server-features.html). [`webdriver.tests.support.inline`](https://github.com/web-platform-tests/wpt/blob/master/webdriver/tests/support/inline.py) lets you pass through a `domain` kwarg to the [`url()` fixture](https://github.com/web-platform-tests/wpt/blob/master/webdriver/tests/support/fixtures.py#L170) to select the site to serve the test document from.

We should also have tests for invalid input, e.g. illegal types, empty string, and for the default behaviour (the absence of `samesite`).

File: webdriver/tests/add_cookie/add.py [hunk 1]
```diff
@@ -78,7 +82,8 @@ def test_add_cookie_for_ip(session, url, server_config, configuration):
         "domain": "127.0.0.1",
         "path": "/",
         "httpOnly": False,
-        "secure": False
+        "secure": False,
+        "sameSite": "Lax"
```
Comment (whimboo) [inline webdriver/tests/add_cookie/add.py]: @christian-bromann could you please add a separate parameterized test for all the available values of same site please? If you don't have the time please let us know and we can cover the test additions with https://bugzilla.mozilla.org/show_bug.cgi?id=1606439.

Also we don't check for invalid values yet, and getCookies also needs a test.

**Review by whimboo — CHANGES_REQUESTED** [review]
Sorry, that it has been taken that long to review, but I wasn't aware of this PR. Please let us know if you have the time to finish this patch off or not. Thanks.

**Comment by whimboo** [comment]
Given that we haven't gotten a reply from  @christian-bromann we have tests implemented when adding geckodriver support. Those will merge  via https://github.com/web-platform-tests/wpt/pull/24136. Marking this PR as  dupe.

**Pull request closed without merging**